FRIEND/Settings.cpp

```diff
@@ -164,7 +166,7 @@ bool Settings::show()
 		}
 		
 		bool procEnabled = checkboxMask & 0x1;
-		bool docEnabled = checkboxMask & 0x2;
+		bool docEnabled = (checkboxMask >> 0x1) & 0x1;
```

Comment:
OK, I can't say I am happy with this change, but I do agree my code is not great either :)
So let me offer another, more readable way:
```
const ushort kCheckBoxMask_ProcEnable = (0b01);
const ushort kCheckBoxMask_DocEnable = (0b10);
...
bool procEnabled = checkboxMask & kCheckBoxMask_ProcEnable;
bool docEnabled = checkboxMask & kCheckBoxMask_DocEnable;
```